styles.css

```diff
@@ -0,0 +1,420 @@
+  font-size: 1rem;
+}
+
+.navbar {
+  border-bottom: 1px solid #e5e5e5;
+  margin: 0;
+}
+
+nav ul li {
+  list-style: none;
+}
+
+main {
+  padding: 2rem;
+}
+
+.featured-col img {
+  width: 100%;
+}
+
+.featured {
+  max-width: 98%;
+  margin: auto;
+}
+
+.fa-search {
+  position: absolute;
+  left: 50%;
+}
+
+.p-relative {
+  position: relative;
+}
+
+.highlights {
+  color: #666;
+}
+
+.article-post h4 {
+  font-weight: bold;
+}
+
+.article-post h4,
+.article-post p {
+  color: #000;
+}
+
+.article-post h4:hover,
+.article-post h5:hover,
+.article-post h6:hover {
+  text-decoration: underline;
+  text-decoration-color: #f72210;
+  color: #000;
+}
+
+.article-img {
+  position: relative;
+}
+
+.sub-title {
+  position: absolute;
+  left: 0;
+  background: rgba(255, 255, 255, 0.75);
+  color: #f72210;
+  padding: 0.1rem 0.5rem;
+  font-weight: bold;
+  bottom: 0;
+}
+
+.advert-title {
+  color: steelblue;
+}
+
+.ad-box {
+  border: 1px solid #e5e5e5;
+  padding: 1rem 1.5rem;
+  margin-bottom: 1rem;
+}
+
+.featured-stories-col {
+  padding: 0 1.5rem;
+}
+
+.post-title {
+  font-weight: bold;
+}
+
+.weekly-mid-sec,
+.subscribe-row {
+  background: #f4f5f6;
+}
+
+.subscribe-box {
+  background: #f4f5f6;
+  border: 1px solid #e5e5e5;
+  padding: 1.5rem;
+}
+
+.p-topics {
+  margin-left: 0.7rem;
+  background: #afe2d2;
+  display: inline-block;
+  font-size: 0.87rem;
+  padding: 0.2rem 0.7rem;
+  font-family: sans-serif;
+}
+
+.p-short-desc {
+  font-weight: 300;
+  color: #666;
+  font-family: sans-serif;
+  margin: 0;
+}
+
+.n-subscribe-btn {
+  border-radius: 0;
+  margin: 1rem 0;
+}
+
+.more-stories .highlights,
+.latest-posts .highlights {
+  padding-bottom: 0.5rem;
+  border-bottom: 1px solid #e5e5e5;
+}
+
+.more-stories .post,
+.latest-posts .post {
+  padding-top: 0.7rem;
+  border-bottom: 1px solid #e5e5e5;
+  padding-bottom: 0.7rem;
+}
+
+.featured-story-col .post {
+  margin-bottom: 1rem;
+}
+
+.in-magazine,
+.editor-picks,
+.featured-slideshows {
+  margin-top: 0.8rem;
+}
+
+.in-magazine .top-title,
+.editor-picks .top-title,
+.featured-slideshows .top-title {
+  font-weight: bold;
+  color: #000;
+  border-bottom: 1px solid #e5e5e5;
+  padding-bottom: 0.8rem;
+}
+
+.see-features {
+  background: #f4f5f6;
+  font-weight: bold;
+  color: #000;
+  padding: 0.5rem 0.8rem;
+}
+
+.in-magazine .top-title img,
+.editor-picks .top-title img,
+.featured-slideshows .top-title img {
+  max-width: 20px;
+  margin-right: 0.4rem;
+  position: relative;
+  top: -3px;
+}
+
+.cat-row {
+  border-bottom: 1px solid #e5e5e5;
+}
+
+.categories {
+  margin: 4rem 0 2rem 0;
+  border-top: 1px solid #e5e5e5;
+  padding: 2rem 0;
+}
+
+.cat-title {
+  color: #f72210;
+  font-weight: bold;
+}
+
+.category-box .third-col .post {
+  padding-top: 0.7rem;
+  border-bottom: 1px solid #e5e5e5;
+  padding-bottom: 0.7rem;
+}
+
+.nw-img {
+  height: 100%;
+  object-fit: cover;
+}
+
+.newsletter-row {
+  border: 1px solid #e5e5e5;
+  max-width: 98%;
+  margin: auto;
```

Comment:
Consider using a `Bootstrap` class name for this selector element and in all other occurrences instead.

`mx-auto` Bootstrap class name can be used instead. See [horizontal centering](https://getbootstrap.com/docs/4.1/utilities/spacing/#horizontal-centering) in bootstrap documentation.

> Remember to use Bootstrap CSS classes wherever it is possible - this is the main point of this project.

#### Please, lets do this for all other occurrences. :blush: